zerver/lib/send_email.py

```diff
@@ -31,6 +31,7 @@ def build_email(template_prefix, to_user_id=None, to_email=None, from_name=None,
         'realm_name_in_notifications': False,
         'support_email': FromAddress.SUPPORT,
         'verbose_support_offers': settings.VERBOSE_SUPPORT_OFFERS,
+        'server_uri': settings.SERVER_URI,
```

Comment:
you mean the email_image_base_uri would be `settings.SERVER_URI + 'static/images/emails/'`?
If not, what would we use for e.g. "create a new realm confirmation" email?

the comment of the top is misleading, I'll fix that as well.